packages/blazor-workspace/BlazorWorkspace.sln

```diff
@@ -9,9 +9,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NimbleBlazor.Tests", "Tests
 EndProject
```

Comment:
Think it's probably worthwhile to have Demo apps for the new execution modes like we do for the acceptance tests (one with mixed interaction modes and one full SSR). Wouldn't need to be additions in this PR

Reply:
#2462